web/src/components/l10n/InstallerKeymapSwitcher.jsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { FormSelect, FormSelectOption } from "@patternfly/react-core";
+
+import cockpit from "../../lib/cockpit";
+
+import { Icon } from "../layout";
+import { _ } from "~/i18n";
+import { useInstallerL10n } from "~/context/installerL10n";
+import { useL10n } from "~/context/l10n";
+import { localConnection } from "~/utils";
+
+const sort = (keymaps) => {
+  // sort the keymap names using the current locale
+  const lang = cockpit.language || "en";
+  return keymaps.sort((k1, k2) => k1.name.localeCompare(k2.name, lang));
```

Comment:
Keymaps should come already translated from D-Bus.

Reply:
I guess you meant "sorted" here...

Yes, they are sorted. But are they sorted using the current locale? I mean in some languages the sorting might be  a bit different than in English...

Reply:
I mean, why do we need `#localeCompare`? Isn't comparison of `keymap.name` values enough?

Reply:
Some languages have special rules for sorting, for example in Estonian the letter `Z` is not the last letter. It is between `S` and `T`. See https://en.wikipedia.org/wiki/Estonian_orthography

We even had some "fun" with that in `[a-z]` regular expressions, see bug [#177560](https://bugzilla.suse.com/show_bug.cgi?id=177560).

I'm not sure if `localectl` output is locale sorted or not, this is just to be really sure.

Reply:
You can even try that in browser console: 
`["t", "a", "z"].sort((k1, k2) => k1.localeCompare(k2, "et"))` returns ` ['a', 'z', 't']`